Projects/05matchGame/index.js

```diff
@@ -0,0 +1,402 @@
+(async function initFunc() {
+  let isNotMobile = window.matchMedia(
+    "only screen and (min-width: 700px)"
+  ).matches;
+  let savedCards = [];
+  let deckId = "";
+  let allSavedCards = [];
+  let allFaceCards;
+  let slots = 16;
+  let cardOne;
+  let cardTwo;
+  let matchFlipCards;
+  let flipCount = 0;
+  let timeCount = 60;
+  let timerFunction;
+  let matchCardsCounter = 0;
+  let maxPairCards = 8;
+  let savedDataFromGame = JSON.parse(localStorage.getItem("gameResults")) || [];
+  let takeResultLocalStorage;
+  let currentResult;
+  let lockCards = false;
+  let newHTMLTable;
+  const imgURLGameOver = "images/gameover.png";
+  const imgURLWin = "images/win.png";
+  const flipAnimationDuration = 1200;
+  const showPopUp = 1000;
+  const imgURLNewRecord = "images/newRecord.jpg";
+  let currentMatchedCards = 0;
+  let result = "";
+  let currentTime = 0;
+
+  let uiElements = {
+    $containerCards: document.querySelector(".containerCards"),
+    $popupContainer: document.querySelector(".popup"),
+    $closePopup: document.querySelector(".close"),
+    $faceCard: document.querySelectorAll(".faceCard"),
+    $flipCounter: document.querySelector(".flipCounter"),
+    $backCard: document.querySelector(".backCard"),
+    $timeCounter: document.querySelector(".timeCounter"),
+    $resetBtn: document.querySelector(".resetBtn"),
+    $gameResults: document.querySelector(".gameResults"),
+    $secondCounter: document.querySelector(".counterSeconds"),
+    $numbersOfFlips: document.querySelector(".counterFlips"),
+    $heightScore: document.querySelector(".heightScoreMsg"),
+  };
+
+  let sysMsg = {
+    popup: document.querySelector(".popup"),
+    popupWinLoseNotResponse: document.querySelector(".sysMsg"),
+    popupImage: document.querySelector(".popupContent"),
+    resultGameOver: "Game Over",
+    resultWin: "Win",
+    notResponseAPI: "Error: Site not found",
+  };
+
+  //check the screen size - the game starts only for screens 700px+;
+  if (isNotMobile) {
```

Comment:
On load the page this will work properly, but when you try to resize the window it's not update the view. So please read this and try to implement it - https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event